File: rest/accounts/list-get-example-1/list-get-example-1.5.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account SID and Auth Token from twilio.com/console
# To set up environmental variables, see http://twil.io/secure
account_sid = ENV['TWILIO_ACCOUNT_SID']
auth_token = ENV['TWILIO_AUTH_TOKEN']

# Initialize Twilio client
@client = Twilio::REST::Client.new(account_sid, auth_token)

# Loop over accounts and print out a property for each one
@client.api.accounts.list.each do |account|
  puts account.date_created
end

File: rest/accounts/list-get-example-1/list-get-example-1.5.x.php
<?php
// Get the PHP helper library from https://twilio.com/docs/libraries/php
require_once '/path/to/vendor/autoload.php'; // Loads the library
use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token = "your_auth_token";
$client = new Client($sid, $token);

// Loop over the list of accounts and echo a property for each one
foreach ($client->api->accounts->read() as $account) {
    echo $account->dateCreated->format('Y-m-d H:i:s');
}
